src/plugins/intel_gpu/include/intel_gpu/plugin/device_config.hpp

```diff
@@ -16,6 +16,8 @@ namespace ov {
 namespace runtime {
 namespace intel_gpu {
 
+const uint32_t default_num_streams_for_tput = 2;
```

Comment:
@myshevts  I did same thing with your PR here

Reply:
sure, again, this is  a plaholder so [when someone implement the GetMetric for the #command-streamers, this var indicates the place where to call that, as we discussed with@vladimir-paramuzov](https://github.com/openvinotoolkit/openvino/pull/7883/commits/b52768c2cc9dbc0aa4ec7c4999de215e3a066b67#diff-a9b4426da3bf04732eec03004bd4351c68573a79f54646420b24189ef1f73e7bR18)